contracts/service-registry/src/migrations/v_0_4.rs

```diff
@@ -88,15 +89,16 @@ mod test {
                 .unwrap();
         }
 
-        migrate_services(&mut deps.storage).unwrap();
+        let coordinator_contract = Addr::unchecked("coordinator");
+        migrate_services(&mut deps.storage, coordinator_contract.clone()).unwrap();
 
         for service in &initial_services {
             let migrated_service: Service =
                 SERVICES.load(&deps.storage, service.name.as_str()).unwrap();
 
             let expected_service = Service {
```

Comment:
```suggestion
            let expected_service = Service {
               coordinator_contract: coordinator_contract.clone(),
               ..service
            };
```

Reply:
this doesn't work, they are different structs and Rust doesn't allow this operation with different structs